<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewAccountWindow</class>
 <widget class="QMainWindow" name="NewAccountWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Smart Wallet</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>391</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>Please fill the required information: </string>
    </property>
    <widget class="QPushButton" name="SaveButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>280</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>361</width>
       <height>221</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="NameLabel">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="NameEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="NewUsernameLabel">
        <property name="text">
         <string>Username</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="NewUsernameEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="NewPasswordLabel">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="NewPasswordEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="AgeLabel">
        <property name="text">
         <string>Age</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="AgeEdit"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="NatIDLabel">
        <property name="text">
         <string>National ID</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="NationalIDEdit"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="AddressLabel">
        <property name="text">
         <string>Address</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="AddressEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="BalanceLabel">
        <property name="text">
         <string>Balance</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="BalanceEdit"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>477</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
